src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+  const RENT_PER_DAY = 40;
+
+  let discount = 0;
+
+  if (days >= SHORT_TERM) {
```

Comment:
You should use `const` instead of `let` for the `discount` variable since its value is not being reassigned. This adheres to the checklist item recommending the use of `const` over `let` where possible to prevent unintentional changes.